core/src/builder/script.rs

```diff
@@ -517,7 +517,7 @@ mod tests {
     }
 
     fn dummy_pubkey() -> PublicKey {
```

Comment:
`unspendable_pubkey` seems more appropriate
```suggestion
    fn unspendable_pubkey() -> PublicKey {
```

Reply:
No, it's dummy so we don't care if it is spendable or not i guess?

Reply:
this is test so it doesn't matter yeah